src/services/upower/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+set_source_files_properties(org.freedesktop.UPower.xml PROPERTIES
+	CLASSNAME DBusUPowerApp
```

Comment:
I'd change this to `DBusUPowerDaemon`, `DBusUPowerServer` or something like that.

Reply:
I agree, I'm going with DBusUPowerService as that may be more suitable.